python/flatnessTask.py

```diff
@@ -34,9 +34,13 @@ def flatnessTask(sensor_id, infile, dtype='OGP', pickle_file=None):
     #
     azims = (10, 45)
     for azim in azims:
-        sensorData.flatness_plot(azim=azim)
+        sensorData.flatness_plot(azim=azim, title='Surface Plot, %s' % sensor_id)
         metData.plot.save('%s_flatness_point_cloud_azim_%i.png'
                           % (sensor_id, azim))
 
     if pickle_file is not None:
         sensorData.persist(pickle_file)
+
```

Comment:
This example code should be removed.   If it's useful for testing the code, then this should be exercised in a test program in a tests folder at the top level of the repo.